plugins/out_opentelemetry/opentelemetry.c

```diff
@@ -295,7 +295,6 @@ static void clear_array(Opentelemetry__Proto__Logs__V1__LogRecord **logs,
 
     for (index = 0 ; index < log_count ; index++) {
         otlp_any_value_destroy(logs[index]->body);
-        flb_free(logs[index]);
```

Comment:
will that now leak the logs[] ? or is cleaning these up covered elsewhere?

Reply:
It was being done on line number 803, this is closed in PR 6720